pyomo/contrib/fbbt/interval.py

```diff
@@ -19,13 +26,22 @@ def mul(xl, xu, yl, yu):
 
 
 def inv(xl, xu):
-    if xl <= 0 and xu >= 0:
-        return -math.inf, math.inf
-    return 1.0/xu, 1.0/xl
+    if xl <= feasibility_tol and xu >= -feasibility_tol:
```

Comment:
Comment would be nice: "If denominator can include 0, then quotient is unbounded."

Reply:
done